ba/js/lesson1/dist/script.dev.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+//Task 1
+var yourName = prompt("Your name..");
+console.log("Hello " + yourName + " !"); //Task 2
+
+var CURRENT_YEAR = 2020;
+var birthYear = prompt("When your were born?   ");
+var age = CURRENT_YEAR - birthYear;
+console.log("You are " + age + " years old!"); //Task 3
+
+var side = prompt("Enter length of side of square");
+console.log("The perimetr equals: ".concat(side * 4)); //Task 4
```

Comment:
you can use it but here it's not okay, just `The perimeter equals: ${side} * 4`. But whether it it for practice it's ok